### DISCUSSION TITLE
Continuous upload if started in the app while app is in the background.

### DISCUSSION THREAD

**Discussion by jimski427** [immich-app/immich#8467] [Feature Request]
### The feature

The past few days I have had some issues with uploading larger files. Then never backed up automatically with the background uploader so I had to manually kick off the upload. When doing so I needed to babysit the upload and make sure that my screen didn't go dark and I could not switch to any other app while doing the upload. If I did, that upload would completely fail and I had to restart it.

The ideal way I would expect this to work is that if you start an upload on the app, it will continuously upload until the current ones are done unless you force closed the app or swiped it out of the recent apps list, switching to another app or the screen going dark should not kill the upload. Alternatively, switching to another app, or the screen going dark could just pause the current upload instead of killing it and needing to start from the beginning. 

Also, the app is v.1.100 build.130, server is 1.100 and I am accessing it Via reverse proxy using NPM.

### Platform

- [ ] Server
- [ ] Web
- [X] Mobile

**Comment by bo0tzz** [comment]
Duplicate of #7287
